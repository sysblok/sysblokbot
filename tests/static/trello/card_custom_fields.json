[
    {
      "id": "field_0",
      "value": {
        "text": "Илья Булгаков"
      },
      "idCustomField": "type_0",
      "idModel": "5e91d48a9e9d947b95fb0f75",
      "modelType": "card"
    },
    {
      "id": "field_1",
      "value": {
        "text": "https://docs.google.com/document/d/DOC1"
      },
      "idCustomField": "type_1",
      "idModel": "5e91d48a9e9d947b95fb0f75",
      "modelType": "card"
    },
    {
      "id": "field_2",
      "value": {
        "text": "Полина Матавина"
      },
      "idCustomField": "type_2",
      "idModel": "5e91d48a9e9d947b95fb0f75",
      "modelType": "card"
    }
]
