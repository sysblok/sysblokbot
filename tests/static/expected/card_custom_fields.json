[
    {
      "id": "field_0",
      "value": "Илья Булгаков",
      "idCustomField": "type_0"
    },
    {
      "id": "field_1",
      "value": "https://docs.google.com/document/d/DOC1",
      "idCustomField": "type_1"
    },
    {
      "id": "field_2",
      "value": "Полина Матавина",
      "idCustomField": "type_2"
    }
]
